src/screens/OccurrenceDetails/constants.ts

```diff
@@ -0,0 +1,70 @@
+export const occurrenceTypeItems = [
```

Comment:
I think it's better to import the dropdown constants from Occurrence screen. It already exists and makes the editing easier.